mkdocs/mkdocs.yml

```diff
@@ -0,0 +1,66 @@
+site_name: mergerfs
+site_description: mergerfs - a featureful union filesystem
+site_url: http://127.0.0.1:8000
+repo_name: GitHub
+repo_url: https://github.com/oregonpillow/oregonpillow.github.io
```

Comment:
i think it would be just `https://github.com/trapexit/mergerfs/` without the trailing mkdocs? I'm not sure the exact deployment settings you're using and i'm also new to github pages. 

![image](https://github.com/user-attachments/assets/d3e50eeb-f0a2-491a-8c48-9c2908315f5b)
where the branch would be `master` and the folder `mkdocs` ?